acme/acme/client.py

```diff
@@ -27,7 +27,11 @@
 # for SSL, which does allow these options to be configured.
 # https://urllib3.readthedocs.org/en/latest/security.html#insecureplatformwarning
 if sys.version_info < (2, 7, 9):  # pragma: no cover
-    requests.packages.urllib3.contrib.pyopenssl.inject_into_urllib3()
+    try:
+        requests.packages.urllib3.contrib.pyopenssl.inject_into_urllib3()
+    except AttributeError:
+        import urllib3.contrib.pyopenssl
```

Comment:
Possibly also `pragma: no cover` comments similar to the ones we use for other [try-import patterns](https://github.com/certbot/certbot/blob/master/acme/acme/dns_resolver.py#L11)